src/Compilers/CSharp/Test/Semantic/Semantics/DynamicTests.cs

```diff
@@ -1305,30 +1357,49 @@ public void TestDynamicMemberAccess_EarlyBoundReceiver_OuterInstance()
             string source = @"
 using System;
 
-public class A
+class A
 {
     public Action<object> F;
     public Action<object> P { get; set; }
     public void M(int x) { }
-  
+
     public class B
     {
         public void Goo()
         {
             dynamic d = null;
             F(d);
             P(d);
-            M(d);
-        }
-    } 
-}";
-            CreateCompilationWithMscorlibAndSystemCore(source).VerifyDiagnostics(
-                // (15,13): error CS0120: An object reference is required for the non-static field, method, or property 'A.F'
-                Diagnostic(ErrorCode.ERR_ObjectRequired, "F").WithArguments("A.F"),
-                // (16,13): error CS0120: An object reference is required for the non-static field, method, or property 'A.P'
-                Diagnostic(ErrorCode.ERR_ObjectRequired, "P").WithArguments("A.P"),
-                // (17,13): error CS0120: An object reference is required for the non-static field, method, or property 'M'
-                Diagnostic(ErrorCode.ERR_ObjectRequired, "M(d)").WithArguments("A.M(int)"));
+            /*<bind>*/M(d);/*</bind>*/
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IInvocationExpression ( ? A.B.()) (OperationKind.InvocationExpression, Type: ?, IsInvalid) (Syntax: 'M(d)')
+  Instance Receiver: IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'M')
```

Comment:
>Instance Receiver: IOperation:  (OperationKind.None, IsInvalid) (Syntax: 'M') [](start = 2, length = 77)

Why are we getting None here? Please open an issue to follow up.

Reply:
Filed https://github.com/dotnet/roslyn/issues/22121